packages/frontend/src/components/LightdashVisualisation/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { DBChartTypes } from 'common';
+import EChartsReact from 'echarts-for-react';
+import React, { FC, RefObject } from 'react';
+import { ChartConfig } from '../../hooks/useChartConfig';
+import SimpleChart from '../SimpleChart';
+import SimpleTable from '../SimpleTable';
+
+interface Props {
+    isLoading: boolean;
+    tableName: string | undefined;
+    chartRef: RefObject<EChartsReact>;
+    chartType: DBChartTypes;
+    chartConfig: ChartConfig;
+}
+
+const LightdashVisualisation: FC<Props> = ({
+    isLoading,
+    tableName,
+    chartRef,
+    chartType,
+    chartConfig,
+}) => (
+    <>
+        {chartType === 'table' ? (
+            <SimpleTable />
+        ) : (
+            <SimpleChart
```

Comment:
Since simple chart can never accept `chartType='table'` we should narrow the type to `Exclude<DBChartType,DBChartType.TABLE>`

This way you won't get the type errors on the switch statements in the `SimpleChart` component